namespaces/cloud-platform-live-0.k8s.integration.dsd.io/cica-apply-for-compensation-dev/01-rbac.yaml

```
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cica-apply-for-compensation-dev-admin
+  namespace: cica-apply-for-compensation-dev
+subjects:
+  - kind: Group
+    name: "github:criminalInjuriesCompensationAuthority"
```

Hi @armoj -- at the moment we only give access to teams that sit under the `ministryofjustice` github organisation. Are you able to get a CICA team setup under that org?

---

There is already a CICA team within the MinistryOfJustice GH. I have requested to be a member.

---

I am now a member of the CICA team in the MoJ org. Is this what was needed from me?

---

@armoj You just need to change that team name to match the one in the MoJ github org. In this case it is:

```suggestion
    name: "github:CICA"
```